packages/stark-ui/src/modules/table/components/table.component.ts

```diff
@@ -953,6 +955,20 @@ export class StarkTableComponent extends AbstractStarkUiComponent implements OnI
 		return columnFilterValue;
 	}
 
+	/**
+	 * Get the position of the filter box of the specified column.
+	 * @param columnName - Name of the column whose filter box position should be retrieved.
+	 * @returns The position of the filter box of the specified column.
+	 */
+	public getColumnFilterPosition(columnName: string): StarkTableColumnFilter["filterPosition"] | undefined {
```

Comment:
Here there is no need to add `| undefined` because it is already part of the types from the `filterPosition` property.

Can you remove it?

Reply:
I think TS was complaining when it wasn't added while I was testing. 🤔

Reply:
I removed it, seems TS is fine with it :-)

Reply:
K, great. 😄